### ISSUE TITLE
Tests: integration test failed

### ISSUE THREAD

**Issue by outcatcher** [outcatcher/anwil#27]
For details see: https://github.com/outcatcher/anwil/actions/runs/4647505335/jobs/8224545692

**Comment by outcatcher** [comment]
Failed test is `TestUserCreate_200`

```
=== CONT  TestRun/TestUserCreate_200
    users_test.go:33: 
        	Error Trace:	/home/runner/work/anwil/anwil/testing/users_test.go:33
        	Error:      	Not equal: 
        	            	expected: 201
        	            	actual  : 400
        	Test:       	TestRun/TestUserCreate_200
```

It seems, data generated for test can be unacceptable for server

**Comment by outcatcher** [comment]
Fixed in #30
